web/src/lib/components/map/LayerItem.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+	import { Layer } from "ol/layer";
+	import Dot from "./Dot.svelte";
+	import Switch from "../Switch.svelte";
+
+	/**
+	 * Object layer reference to this layerItem
+	 */
+	export let layer: Layer;
+
+	/**
+	 * Switch value
+	 */
+	let visible: boolean = true;
+
+	/**
+	 * Changes layer visibility based on @visible value
+	 */
+	function toggleVisibility() {
+		visible = !visible;
+		layer.setVisible(visible);
+	}
+</script>
+
+<div class="layer-item">
+	{#if layer.get("layer-color")}
+		<Dot color={layer.get("layer-color")} />
+	{:else}
+		<Dot color={"white"} />
+	{/if}
+
+	{#if layer.get("layer-name")}
+		<h2>{layer.get("layer-name")}</h2>
+	{:else}
+		<h2>#UNDEFINED</h2>
```

Comment:
This should be removed since we don't want a #UNDEFINED to be displayed in the UI